WSL/tutorials/wsl-database.md

```diff
@@ -113,18 +113,17 @@ To work with with PostgreSQL databases in VS Code, try the [PostgreSQL extension
 
 To install MongoDB (version 5.0) on WSL (Ubuntu 20.04):
 
-1. Open your WSL terminal (ie. Ubuntu) and go to the root directory: `cd ~`
+1. Open your WSL terminal (ie. Ubuntu) and go to your home directory: `cd ~`
 2. Update your Ubuntu packages: `sudo apt update`
```

Comment:
There is another `sudo apt-get update` in Step 5. Seems like we don't need this one?